README.md

```diff
@@ -90,6 +90,7 @@ avatarUr(not required): string,
 ```
 {
   instructor_id: INCREMENT (primary key, auto-increments, generated by database),
+  instructor_name: STRING (required),
```

Comment:
Glad you've updated the documentation here. That could have been easy to overlook